_assets/javascripts/carousel.js

```diff
@@ -22,9 +22,10 @@ $(document).ready(function () {
     ]
   });
 
-  var today = new Date();
+  var today    = new Date();
+  var firstDay = 4;
 
-  weekday = today.getDate() > 3 ? today.getDay() : 0;
+  weekday = today.getDate() > firstDay ? today.getDay() : 0;
```

Comment:
The `firstDay` variable it is not a bad idea. But if you want to do that you should do `>=`.

Anyway, this doesn't solve the slider starting at Tuesday. The #55 does.

Reply:
I reopened this PR because I think #55 doesn't totally solve the problem.

The first problem is related with the fact that `weekday = today.getDate() > 3 ? today.getDay() - 1 : 0` never reaches the else member when the event starts after the day 3, which happens this edition for instance. So, this is the same as doing `weekday = today.getDay() - 1`.

This solution actually works when the event happens from Monday to Saturday, but when there's a Sunday involved or the event lasts more than one week, #55 is not a good solution since `getDay` returns 0 for Sundays, so `initialSlider = weekday = -1`.

So, I made these slightly deeper changes in this function.

Reply:
I knew that it was a temporary fix. I just closed this because the previous solution didn't solve the problem for this year too. 

The new commit is a much better solution :rocket: